rsts/getting_started.rst

```diff
@@ -1,109 +1,341 @@
+#. Flyte can be deployed locally using a single Docker container — we refer to this as the ``flyte-sandbox`` environment. You can also run this getting started against a hosted or pre-provisioned environment.
 
-10. Give it a minute and once it's done, check out "Inputs/Outputs" on the top right corner to see your updated greeting.
+   .. tabs::
+
+      .. tab:: Start a new sandbox cluster
+
+        .. tip:: Want to dive under the hood into flyte-sandbox, refer to the guide `here<>`_.
+
+        .. prompt:: bash $
+
+           flytectl sandbox start --sourcesPath <full-path-to-myflyteapp>
+
+      .. tab:: Connect to an existing Flyte cluster
+
+        .. prompt:: bash $
+
+            flytectl init
+
+
+.. _getting-started-build-deploy:
+
+2. Build & Deploy Your Application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#. Flyte uses Docker containers to package the workflows and tasks and sends them to the remote Flyte cluster. Thus, there is a ``Dockerfile`` already included in the cloned repo. You can build the Docker container and push the built image to a registry.
+
+   .. tabs::
+
+       .. tab:: Flyte Sandbox
+
+           Since ``flyte-sandbox`` runs locally in a Docker container, you do not need to push the Docker image. You can combine the build and push step by simply building the image inside the Flyte-sandbox container. This can be done using the following command:
+
+           .. prompt:: bash $
+
+               flytectl sandbox exec -- docker build . --tag "myapp:v1"
+
+           .. tip::
+            #. Why are we not pushing the Docker image? Want to understand the details — refer to guide `here <>`_
+            #. *Recommended:* Use the bundled `./docker_build_and_tag.sh`. It will automatically build the local Dockerfile, name it and tag it with the current git-SHA. This helps in achieving GitOps style workflows.
+
+       .. tab:: Remote Flyte Cluster
+
+           If you are using a remote Flyte cluster, then you need to build your container and push it to a registry that is accessible by the Flyte Kubernetes cluster.
+
+           .. prompt:: bash $
+
+               docker build . --tag registry/repo:version
+               docker push registry/repo:version
+
+#. Next, package the workflow using the ``pyflyte`` cli bundled with Flytekit and upload it to the Flyte backend. Note that the image is the same as the one built in the previous step.
+
+   .. prompt:: bash (venv)$
+
+      pyflyte --pkgs myapp.workflows package --image myapp:v1
+
+#. Upload this package to the Flyte backend. We refer to this as ``registration``.
+
+   .. prompt:: bash $
+
+      flytectl register files -p flytesnacks -d development -a flyte-package.tgz  -v v1
+
+#. Finally, visualize the registered workflow.
+
+   .. prompt:: bash $
+
+      flytectl get workflows -p flytesnacks -d development myapp.workflows.example.my_wf -o doturl
```

Comment:
It should be `flytectl get workflows -p flytesnacks -d development myapp.workflows.example.hello_world --version=v1 -o doturl`

Reply:
no it is my_wf https://github.com/flyteorg/flytekit-python-template/blob/main/myapp/workflows/example.py#L44